enhancements/agentic-platform-agent-images/README.md

```
@@ -0,0 +1,494 @@
+| `/opt/skills/` | Baked into the image | Fallback |
+
+If a skill with the same name exists in both locations, the ImageVolume
+mount takes precedence. The harness uses `skillctl` to read skill
+metadata (`skill.yaml`) from each location to determine type (skill vs
+rule) and other properties.
+
+#### Output Convention
+
+The harness writes results to `/.konveyor/` on the workspace PVC:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)